### PR TITLE
Unify code for evaluation

### PR DESCRIPTION
**Pull request by dniku**

### CODE REVIEW

File: week04_approx_rl/seminar_pytorch.ipynb [hunk 1]
```diff
@@ -354,26 +352,20 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# show video\n",
+    "# Show video. This may not work in some environments. If it doesn't\n",
```
Comment (yhn112) [inline week04_approx_rl/seminar_pytorch.ipynb:355]: "environments" doesn't mean "gym environments" here, does it? Somewhat confusing.

**Comment by yhn112** [comment]
I'm sorry to say this, but we need a similar change in coursera branch.

**Comment by dniku** [comment]
BTW, in Coursera, it will require a corresponding change in graders. Luckily, it shouldn't break anything, as in Coursera we can overwrite neither notebooks nor graders that are already deployed. The only potential problem will be if someone wants to reset their notebook to the original state and does the `?forceRefresh=true` trick only with the notebook, but not with the grader. I'm not sure if we should include a workaround in the notebook for that case.

**Comment by dniku** [comment]
#388, but it requires testing.

**Comment by yhn112** [comment]
Looks mergeable, doesn't it?

**Pull request merged**